_knitr/A Shiny app for Hong Kong District Councillors_20200801.md

```diff
@@ -8,43 +8,160 @@ output:                    # DO NOT CHANGE
 
-## The App
+[^1]: There are many reasons for this, and arguably a similar phenomenon can be observed in most local elections in other countries. See Lee, F. L., & Chan, J. M. (2008). Making sense of participation: The political culture of pro-democracy demonstrators in Hong Kong. *The China Quarterly*, 84-101.
 
-<Describe the layout of the app>
+As of the summer of 2020, we identified that 96% (434) of the 452 District Councillors elected in 2019 actually have a dedicated Facebook page for delivering updates to and engaging with local residents. However, these Facebook pages have to be manually searched for online, and there is not a readily available tool where people can quickly map a District to a District Councillor and to their Facebook feeds. 
 
-## Data Collection and Cleaning
-<Insert screenshot>
-- Where the data came from
-- Deliberated to using Google Sheets to avoid bulking up the repo, and also enable participation from those who do not code
+As a wise person once said, "If you can solve a problem effectively in R, why the hell not?" We tackled this problem by creating a Shiny app in R, which brings together the Facebook feeds and constituency information for Hong Kong's district councillors in one place. In this way, people will be able to access the currently disparately stored information in a single web app.
+
+You can access:
+
+- The Shiny app [here](https://hkdistricts-info.shinyapps.io/dashboard-hkdistrictcouncillors/). 
+- Our GitHub repository [here](https://github.com/avisionh/dashboard-hkdistrictcouncillors/). 
+- Don't forget to also provide some feedback to the Shiny app [here](https://hkdistrictsinfo.typeform.com/to/gFHC02gE)!
+
+Whether you are more of an R enthusiast or simply someone who has an interest in Hong Kong politics (hopefully both!), we hope this post will bring you some inspiration on how you can use R _not just_ as a great tool for data analysis, but also as an enabler for you to do something tangible for your community and contribute to causes you care about. 
+
+## 🔍 What is in the app?
+
+<img src="https://raw.githubusercontent.com/martinctc/blog/master/images/hkdc-app-example.png" width=500>
+
+The Shiny app is built like a dashboard which combines information about each district councillor alongside their Facebook page posts (if it exists) and the district they serve, illustrated on an interactive map.
+
+Specifically, there are several key components that were used on top of the incredible [shiny](https://github.com/rstudio/shiny) package:
+
+- [shinydashboard](https://github.com/rstudio/shinydashboard): For mobile-friendly dashboard layout.
+    + We understood that our users, primarily HK citizens, frequently use mobiles. Thus, to ensure this app was useful to them, we centred our design on how the app looked on their mobile browsers.
+    
+<img src="https://raw.githubusercontent.com/martinctc/blog/master/images/googlesheets4.png" width=200>
```

Comment:
A bit weird that we are including the hex sticker for the `googlesheets4` package but not the other ones.

Reply:
We can be comprehensive

Reply:
Not all of the hex stickers are available for the packages - I'll make a call to see whether it looks fine when it's in the Jekyll site template (it looks a bit different than this 'preview' version on GitHub)!